### PR TITLE
Added localization files for brazilian portuguese (#431)

### PR DESCRIPTION
**Pull request by thiagoromanos**
Signed-off-by: Thiago Romano <thiagoromano.s@gmail.com>

# Summary of Changes

Added the localization files needed to enable the brazilian portuguese language in Settings.

# Related Issues

https://github.com/hyperledger/aries-mobile-agent-react-native/issues/431

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [X] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [X] Updated documentation as needed for changed code and new or modified features;
- [X] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/navigators/RootStack.tsx [hunk 1]
```diff
@@ -183,8 +183,8 @@ const RootStack: React.FC<RootStackProps> = (props: RootStackProps) => {
           {(props) => (
             <Onboarding
               {...props}
-              nextButtonText={'Next'}
-              previousButtonText={'Back'}
+              nextButtonText={t('Global.Next')}
```
Comment (JamesKEbert) [inline core/App/navigators/RootStack.tsx:186]: Note, this will mean we are missing "Next" in English & French (so those should be added)

**Review by JamesKEbert — APPROVED** [review]
This looks good to me, although I do say I don't speak Portuguese, so if someone is able to review that does it might be good. 
Thanks for the contribution!

File: core/App/navigators/RootStack.tsx (hunk 1 above)
Comment (thiagoromanos) [reply core/App/navigators/RootStack.tsx:186]: The Global.Next and Global.Back for english and french are already there, but they weren't used in this screen (it was a hard-coded string)

**Comment by thiagomv-ca** [comment]
> This looks good to me, although I do say I don't speak Portuguese, so if someone is able to review that does it might be good. Thanks for the contribution!

Hi, I am Brazilian and a native speaker of Portuguese and I did a review and approved

**Comment by cvarjao** [comment]
Just need rebase and  it should be good to merge

**Comment by thiagoromanos** [comment]
Just rebased the branch. I don't have write permission, so one of you guys will have to merge it for me.

File: core/App/navigators/RootStack.tsx (hunk 1 above)
Comment (JamesKEbert) [reply core/App/navigators/RootStack.tsx:186]: Ahh wonderful I didn't see that, thanks for updating :)

**Pull request merged**